org.eclipse.acute/src/org/eclipse/acute/dotnetnew/DotnetNewWizardPage.java

```diff
@@ -217,15 +238,53 @@ public void createControl(Composite parent) {
 		templateViewer = new ListViewer(list);
 		templateViewer.setContentProvider(new ArrayContentProvider());
 		templateViewer.setComparator(new ViewerComparator()); // default uses getLabel()/toString()
-		templateViewer.add("Loading templates");
-		templateViewer.getList().setEnabled(false);
 		templateViewer.addSelectionChangedListener(e -> {
 			setPageComplete(isPageComplete());
 		});
 		templateControlDecoration = new ControlDecoration(templateViewer.getControl(), SWT.TOP | SWT.LEFT);
 		templateControlDecoration.setImage(errorImage);
+		updateTemplateList();
+
+		//Update Template List with preferences change
+		IPreferenceStore store = AcutePlugin.getDefault().getPreferenceStore();
+		store.addPropertyChangeListener(new IPropertyChangeListener() {
```

Comment:
You'll need to remove this listener later of this can introduce a memory leak (the listener never gets disposed nor garbage collected). So let's make it a local var or a field and remove it when the main composite, dialog or page is disposed.